src/overlays/actors/ovl_En_Mb/z_en_mb.c

```diff
@@ -1007,23 +1001,20 @@ void func_80AA87D8(EnMb* this, GlobalContext* globalCtx) {
         if (this->unk_32A == 0) {
             if (Math_Rand_ZeroOne() > 0.7f) {
                 this->unk_32A = Math_Rand_S16Offset(50, 70);
-                this->soundTimer = Math_Rand_S16Offset(26, 40) & 0xFFFF;
+                this->soundTimer = Math_Rand_S16Offset(15, 40);
             } else {
                 func_80AA6830(this);
             }
         }
     }
     if (currentFrame != (s32)this->skelAnime.animCurrentFrame) {
-        if (!((sp48 >= 2) || (((s32)playbackSpeedABS + (s32)this->skelAnime.animCurrentFrame) <= 0)) ||
+        if (((sp48 <= 1) && ((s32)playbackSpeedABS + currentFrame >= 1)) ||
             ((sp48 < 21) && (((s32)playbackSpeedABS + currentFrame >= 20)))) {
```

Comment:
Very minor but we might as well be consistent
```suggestion
            ((sp48 <= 20) && (((s32)playbackSpeedABS + currentFrame >= 20)))) {
```

Reply:
Huh, I thought I did that.